tests/python/unittest/test_operator.py

```diff
@@ -4372,6 +4347,45 @@ def get_large_matrix():
                            expected=[gt_topk(dat=a_npy, axis=1, ret_typ="indices", k=1,
                                              is_ascend=True)])
 
+    for dtype in [np.float16, np.float32, np.float64]:
```

Comment:
Would you also test for `int32` and `int64`? Also, the testing for `uint8` and `int8` can be added separately.

Reply:
it is already test in `test_ndarray.py` here - https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_ndarray.py#L908